src/ui/drawing/gdi/GDIBitmapSurface.cpp

```diff
@@ -147,6 +147,7 @@ void GDIAlphaBitmapSurface::Blend(HDC hTargetDC, int nX, int nY) const noexcept
 
     UINT32* pBits;
     HBITMAP hBitmap = CreateDIBSection(hMemDC, &bmi, DIB_RGB_COLORS, reinterpret_cast<LPVOID*>(&pBits), nullptr, 0);
+    Expects(hBitmap != nullptr);
     SelectBitmap(hMemDC, hBitmap);
```

Comment:
Example of not adhering to specification. In `SelectObject`, the `HDIOBJ `is an "in" param but `hBitmap `could be a `nullptr`, so we have to make it a requirement that it's not a `nullptr`.